lib/block.rb

```diff
@@ -2,65 +2,23 @@
 
 require_relative './helpers.rb'
 require 'openssl'
-require 'pg'
 
-# Understands creating and retrieving new blocks
+# block class
 class Block
-  attr_reader :id, :receiver
+  attr_reader :transactions, :previous_hash, :timestamp, :hash
 
-  def initialize(id, data, hash)
-    @id = id
-    @sender = data[:sender]
-    @receiver = data[:receiver]
-    @value = data[:value]
-    @previous_tx = data[:previous_tx]
-    @hash = hash
+  def initialize(transactions:, previous_hash:, timestamp: Time.now)
```

Comment:
What's the reason behind setting these parameters as symbols in the list of arguments? Not seen this done before, is it a good convention?